<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogpidlist</class>
 <widget class="QDialog" name="dialogpidlist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="table_pid">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>261</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>PID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Count</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>%</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Desctiption</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_packet_count">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_pid_used">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
